templates/taxi/car_detail.html

```diff
@@ -3,20 +3,33 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    <form method="post" action="">
+      {% csrf_token %}
+      <button type="submit"
+          {% if request.user in car.drivers.all %} class="btn btn-danger" {% else %}
+              class="btn btn-success" {% endif %}>
+        {% if request.user in car.drivers.all %}
+          Delete me from this car
+        {% else %}
+          Assign me to this car
+        {% endif %}
+      </button>
+    </form>
+  </h1>
```

Comment:
Code Style: It seems like your HTML indentation is not consistent. As per the checklist, you should use 2 whitespaces for indentation in your `.html` files.